lib/Twig/Environment.php

```diff
@@ -853,7 +883,7 @@ public function setExtensions(array $extensions)
     /**
      * Returns all registered extensions.
      *
-     * @return array An array of extensions
+     * @return array An array of extensions (keys are for internal usage only and should not be relied on)
```

Comment:
To make the intent more clear `array` could be replaced with `Twig_ExtensionInterface[]`.